scripts/repository.lic

```diff
@@ -1382,10 +1403,46 @@ elsif (cmd[0] =~ /^download$/i)
   if list.empty?
     echo "error: file not found"
   elsif list.length > 1
+    names = list.map { |row| row[0] }
     list.unshift(headers)
-    respond "\nMultiple files match: be more specific\n\n#{format_list.call(list)}\n\n"
+    puts "\nMultiple files match: be more specific\n\n#{format_list.call(list, 'download')}\n\n"
```

Comment:
do NOT use `puts` as it has known issues with terminal based FEs. In order to send actual XML, change the `respond` to `_respond`